app/[lang]/chapters/[slug]/[lesson]/page.tsx

```diff
@@ -112,10 +115,19 @@ export default function Page({ params }) {
     return navigation.redirect(lastUnlockedLessonPath)
   }
 
+  function handleChange(evt) {
+    setVal(evt.target.value)
+  }
+
   return (
     unlocked && (
       <>
         <Head />
+        <NumericInput
+          label={'string'}
+          value={val}
+          onChange={(e) => setVal(e.target.value)}
```

Comment:
I think `e.target.value` will actually be a string in some browsers, we need to make sure it returns a number (see prev comments)

Reply:
Addressed in 1c7cd6cb913ce1305fa02df9af0a4e5b9c955001
The latest implementation passes the setValue function in the NumericInput.